### PR TITLE
Authenticated and withUser components

### PR DESCRIPTION
**Pull request by TimMikeladze**

### CODE REVIEW

File: src/Authenticated.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, { PropTypes } from 'react';
+import { AccountsClient } from '@accounts/accounts';
+import FormTypes from './FormTypes';
+
+class Authenticated extends React.Component {
```
Comment (alexyuly) [inline src/Authenticated.js]: You could `import { Component } from 'react'`

File: src/Authenticated.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  static propTypes = {
```
Comment (alexyuly) [inline src/Authenticated.js]: Just curious, any particular reason you're using `static` here?

Comment (TimMikeladze) [inline src/Authenticated.js]: Do you prefer `Authenticated.propTypes`?

Comment (alexyuly) [inline src/Authenticated.js]: Yeah, because it is what we are using elsewhere. But if this is the style you prefer in this package, it is fine.

Comment (TimMikeladze) [inline src/Authenticated.js]: I prefer using `static` when using a class.

Comment (alexyuly) [inline src/Authenticated.js]: I do not mind that at all.

**Pull request merged**